holoviews/plotting/plot.py

```diff
@@ -243,6 +247,15 @@ def __init__(self, keys=None, dimensions=None, layout_dimensions=None,
         self._force = False
         self._updated = False # Whether the plot should be marked as updated
 
+        # Setting up alias
+        if 'title' in params and 'title_format' in params:
+            if params['title'] != params['title_format']:
+                self.warning('The title and title_format parameters do not match. Using title.')
+        elif 'title_format' in params:
+            params['title'] = params['title_format']
+        elif 'title' in params:
+            params['title_format'] = params['title']
```

Comment:
This is not the right place to do these checks since it can be set dynamically and title format would currently be ignored. Generally that means doing this where the parameter is actually used and not altering the actual parameter values on the plot.